src/libsync/discoveryphase.cpp

```diff
@@ -246,6 +246,11 @@ void DiscoveryPhase::scheduleMoreJobs()
     }
 }
 
+bool DiscoveryPhase::isSpace() const
+{
+    return !(Utility::urlEqual(_account->davUrl(), _baseUrl) || _account->davUrl().isParentOf(_baseUrl));
```

Comment:
Please have another look, it returned true before in the case that both strings where equal

Reply:
Seems okay now.